src/xcode/ENA/ENA/Source/AppDelegate & Globals/NotificationManager.swift

```diff
@@ -34,8 +34,12 @@ final class NotificationManager: NSObject, UNUserNotificationCenterDelegate {
 		if notification.request.identifier.contains(LocalNotificationIdentifier.checkout.rawValue) {
 			eventCheckoutService.checkoutOverdueCheckins()
 		}
-
-		completionHandler([.alert, .badge, .sound])
+		
+		if #available(iOS 14.0, *) {
```

Comment:
The commit title talks about iOS 15 but here it says iOS 14, just wanted to make sure that iOS 14 is correct here?

Reply:
this if is about the iOS 14 SDK - the code inside the if is only available in iOS 14 or later.

Reply:
Okay, then the title of the commit was a bit misleading. Thank you @KaiTeuber!